src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -36,22 +38,30 @@
 public class ActorsController {
     @Autowired
     ActorsDao actorsDao;
```

Comment:
Please put space between two variables that have decorations.

Reply:
Good point, will add it as part coding conventions task